### DISCUSSION TITLE
因为字体原因thuthesis-master无法编译

### DISCUSSION THREAD

**Discussion by tiany97** [tuna/thuthesis#917] [Q&A] [answered]
**mac上下载thuthesis-master后解压，输入make thesis后无法编译，以下为报错**

./thuthesis.cls:1054: Package fontspec Error: The font "KaiTi" cannot be found.

For immediate help type H <return>.
 ...                                              
                                                  
l.1054 \thu@set@cjk@font
                        
No pages of output.
Transcript written on thuthesis-example.log.
Latexmk: Getting log file 'thuthesis-example.log'
Latexmk: Examining 'thuthesis-example.fls'
Latexmk: Examining 'thuthesis-example.log'
Latexmk: Log file says no output from latex
Latexmk: For rule 'xelatex', no output was made
Latexmk: Errors, so I did not complete making targets
Collected error summary (may duplicate other messages):
  xelatex: Command for 'xelatex' gave return code 1
      Refer to 'thuthesis-example.log' for details
Latexmk: If appropriate, the -f option can be used to get latexmk
  to try to force complete processing.
make: *** [thuthesis-example.pdf] Error 12

**但是我的windows路径“/Applications/Microsoft Word.app/Contents/Resources/DFonts”下是有Kaiti.ttf这个文件的

求问这么解决**

**Comment by tiany97** [comment]
解决了。可能是mac的问题。
把字体从Applications/Microsoft Word.app/Contents/Resources/DFonts复制到桌面然后单击安装后就可以编译了

**Reply by zepinglee** [reply, marked as answer]
应该是 #913 相同的问题。
